src/main/kotlin/com/devfloor/untitled/course/domain/Course.kt

```diff
@@ -99,7 +99,7 @@ class Course(
         this.board = board
     }
 
-    fun isSame(course: Course): Boolean =
+    fun matchRequiredInfo(course: Course): Boolean =
```

Comment:
이 메서드 네이밍 이렇게 하는게 어때??
```suggestion
    fun matchesRequiredInfo(course: Course): Boolean =
```

Reply:
이거 match랑 matches랑 어떤 차이야?? 내가 영어가 좀 부족해서...ㅋㅋㅋㅋ ㅠㅠ

Reply:
`match~~` 라는 네이밍이 뭔가 명령형이니까 딱 봤을때 불리언 함수가 아닌거같아서 !?
보통 불리언 함수는 `is~~` `has~~` `does~~~` 이렇게 많이 쓰니까 여기도 네이밍을 그렇게 맞추는게 좋을거같아서 !